commons/src/main/java/com/powsybl/commons/extensions/Extension.java

```diff
@@ -33,4 +33,12 @@
      */
     void setExtendable(T extendable);
 
+    /**
+     * Return whether the extension is empty (and thus should not be serialized)
+     *
+     * @return true if the extension is empty, false otherwise
+     */
+    default boolean isEmpty() {
+        return false;
+    }
```

Comment:
This method sounds very strange. I'm not convinced this method is well placed. For serialization purpose, why the serializer is not responsible to determine if an extension should be serialized or not?

Reply:
Yes indeed, that would be more consistent...